charts/kube-master/templates/api.yaml

```diff
@@ -189,7 +191,9 @@ spec:
             - --enable-admission-plugins=PersistentVolumeLabel
 {{- end }}
 {{- if (semverCompare ">= 1.14" .Values.version.kubernetes) }}
+{{- if (semverCompare "< 1.17" .Values.version.kubernetes) }}
```

Comment:
Interesting, why is this necessary?
This might conflict with hammertime which depends on frequent node updates to not kick in. 
In general we should switch to NodeLeases anyway but this requires updating the client-go in kubernetes.

Reply:
Apparently this feature needs to be turned on:
```
error: invalid argument "NodeLease=false" for "--feature-gates" flag: cannot set feature gate NodeLease to false, feature is locked to true
```